doc/development-process.md

```diff
@@ -0,0 +1,132 @@
+# Development Process
+
+Depending on your area of focus we're providing guidance of each kind of development setup you might want.
+
+Step one, you **must** have the following things installed:
+
+* [go](https://go.dev) v1.17 -- Primary development language
+* [docker](https://www.docker.com/) -- Used for generating containers & testing kubernetes set-ups.
+* [kind](https://kind.sigs.k8s.io/) -- Run kubernetes clusters in docker for testing.
+* [flux](https://fluxcd.io/) -- Continuous delivery system for kubernetes that weave-gitops enriches.
+* [tilt](https://tilt.dev/) -- Automatically build and deploy to a local cluster.
+
+Now you have those things installed, lets go!
+
+## Quick start development setup
+
+Help me get going as fast as possible!
+
+Create a kind cluster. We'll do this so we can have a local cluster on our machine
+
+`$ kind create cluster --name=<some name>` 
+
+Then we install flux on it.
+
+`$ flux install`
+
+Then we bring up tilt.
+
+`$ tilt up`
```

Comment:
So trying a fresh clone on the repo and the steps up to this point I then hit this error:
```

(Tiltfile)
<30s ago
Update error
Filter by text or /regexp/
​
Filter resource logs by text or /regexp/
Initial Build • (Tiltfile)
Loading Tiltfile at: /Users/harris/Projects/wgo-v2/weave-gitops/Tiltfile
local: ./tools/bin/helm template dev ./charts/gitops-server -f ./tools/helm-values-dev.yaml
 → sh: ./tools/bin/helm: No such file or directory
Traceback (most recent call last):
  /Users/harris/Projects/wgo-v2/weave-gitops/Tiltfile:42:19: in <toplevel>
  /Users/harris/Projects/wgo-v2/weave-gitops/Tiltfile:40:14: in helmfiles
  <builtin>: in local
Error: command "./tools/bin/helm template dev ./charts/gitops-server -f ./tools/helm-values-dev.yaml" failed.
error: exit status 127
```

Reply:
is `make all` (or other) requested somewhere else?

Reply:
oops, sorry. Ok, I'll take a look thanks.

Reply:
in my simplified instructions we're not going to do that, I was missing a step of install helm. It's been updated.

Reply:
works for me now :+1: